object_detection_tracking/include/object_detection_tracking_worker.h

```diff
@@ -1,3 +1,5 @@
+#pragma pack
```

Comment:
You still have include guards in this header. I am saying to use #pragma OR include guards NOT both.

Reply:
Check your other headers as well